cmd/openshift-install/migrate/azure/migrate.go

```diff
@@ -0,0 +1,45 @@
+package azure
+
+import (
+	"github.com/spf13/cobra"
+
+	azmigrate "github.com/openshift/installer/pkg/migrate/azure"
+)
+
+var (
+	azureMigrateOpts struct {
+		zone              string
+		oldResourceGroup  string
+		newResourceGroup  string
+		virtualNetwork    string
+		vnetResourceGroup string
+		link              bool
+	}
+)
+
+func runMigrateAzurePrivateDNSMigrateCmd(cmd *cobra.Command, args []string) error {
+	if (azureMigrateOpts.zone == "" || azureMigrateOpts.oldResourceGroup == "") || (azureMigrateOpts.link == true && azureMigrateOpts.virtualNetwork == "") || (azureMigrateOpts.virtualNetwork != "" && azureMigrateOpts.vnetResourceGroup == "") {
+		return cmd.Help()
+	}
+
+	return azmigrate.Migrate(azureMigrateOpts.oldResourceGroup, azureMigrateOpts.newResourceGroup, azureMigrateOpts.zone, azureMigrateOpts.virtualNetwork, azureMigrateOpts.vnetResourceGroup, azureMigrateOpts.link)
+}
+
+// NewMigrateAzurePrivateDNSMigrateCmd adds the migrate command to openshift-install
+func NewMigrateAzurePrivateDNSMigrateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "migrate",
```

Comment:
so this currently is `openshift-install migrate azure privatedns migrate` ? now that is sounding really bad. can we improve this a little bit...